libs/designsystem/src/lib/scss/print/elements/_button.scss

```diff
@@ -0,0 +1,6 @@
+@media print {
+  // Buttons do not make much sense on print, but this may be too far reaching
```

Comment:
I agree with your comment. I think it is very context dependent & should be handled by the consumer projects.